documentation/course/e2/README.md

```diff
@@ -681,36 +689,47 @@ TODO @jakubzehner
     <th>Wady</th>
   </tr>
   <tr>
-    <th>Rozwiązanie 1</th>
+    <th>Wspólna baza danych dla mikroserwisów</th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Prostsze zarządzanie danymi</li>
+        <li>Łatwe współdzielenie danych</li>
+        <li>Mniejsza złożoność systemu</li>
       </ul>
     </td>
     <td>
       <ul>
-        <li>Wada 1</li>
+        <li>Słabe oddzielenie odpowiedzialności</li>
+        <li>Trudniejsze skalowanie</li>
+        <li>Ryzyko wystąpienia wąskiego gardła</li>
       </ul>
     </td>
   </tr>
   <tr>
-    <th>Rozwiązanie 2</th>
+    <th>Oddzielne bazy dla mikroserwisów</th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Silne oddzielenie odpowiedzialności</li>
+        <li>Łatwiejsze skalowanie</li>
+        <li>Lepsza niezawodność</li>
       </ul>
     </td>
     <td>
       <ul>
-        <li>Wada 1</li>
+        <li>Złożona infrastruktura</li>
+        <li>Trudniejszy dostęp do danych między serwisami</li>
       </ul>
```

Comment:
**suggestion:** I'd mention the inability to do `JOIN`s cross-microservice, which I personally think is one of the biggest downsides to splitting the DB.